tests/streaming/models/test_transcript.py

```diff
@@ -0,0 +1,96 @@
+                sender=Sender.HUMAN,
+                text="Send me an email you bot. My email is du@de.com",
+            ),
+            ActionStart(
+                action_type="action_nylas_send_email",
+                action_input=TwilioPhoneCallActionInput(
+                    action_config=NylasSendEmailActionConfig(),
+                    conversation_id="123",
+                    params=NylasSendEmailParameters(
+                        recipient_email="du@de.com",
+                        body="What up",
+                        subject="This is the bot",
+                    ),
+                    _user_message_tracker=asyncio.Event(),
+                    twilio_sid="123",
+                ),
+            ),
+            ActionFinish(
+                action_type="action_nylas_send_email",
+                action_output=ActionOutput(
+                    action_type="action_nylas_send_email",
+                    response=NylasSendEmailResponse(success=True),
+                ),
+            ),
+        ]
+    )
+
+    assert (
+        transcript.to_string()
+        == """BOT: What up
+HUMAN: Send me an email you bot. My email is du@de.com
+ACTION_WORKER: action_config=NylasSendEmailActionConfig() conversation_id='123' params=NylasSendEmailParameters(recipient_email='du@de.com', body='What up', subject='This is the bot') twilio_sid='123'
+ACTION_WORKER: action_type='action_nylas_send_email' response=NylasSendEmailResponse(success=True)"""
+    )
+
+
+def test_transcript_to_string_no_phone_input():
+    transcript = Transcript(
+        event_logs=[
+            Message(sender=Sender.BOT, text="What up"),
+            Message(
+                sender=Sender.HUMAN,
+                text="Send me an email you bot. My email is du@de.com",
+            ),
+            ActionStart(
+                action_type="action_nylas_send_email",
+                action_input=ActionInput(
+                    action_config=NylasSendEmailActionConfig(),
+                    conversation_id="123",
+                    params=NylasSendEmailParameters(
+                        recipient_email="du@de.com",
+                        body="What up",
+                        subject="This is the bot",
+                    ),
+                    _user_message_tracker=asyncio.Event(),
+                ),
+            ),
+            ActionFinish(
+                action_type="action_nylas_send_email",
+                action_output=ActionOutput(
+                    action_type="action_nylas_send_email",
+                    response=NylasSendEmailResponse(success=True),
+                ),
+            ),
+        ]
+    )
+
+    assert (
+        transcript.to_string()
+        == """BOT: What up
+HUMAN: Send me an email you bot. My email is du@de.com
+ACTION_WORKER: action_config=NylasSendEmailActionConfig() conversation_id='123' params=NylasSendEmailParameters(recipient_email='du@de.com', body='What up', subject='This is the bot')
```

Comment:
can we make the inner objects dictionaries?

Reply:
you want the line to be this?
```
ACTION_WORKER: action_config={} conversation_id='123' params={"recipient_emai": 'du@de.com', "body": 'What up', "subject": 'This is the bot'}
```

Reply:
yes, also let's skip `action_config` and `conversation_id`